mappings/net/minecraft/world/gen/feature/MineshaftFeature.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS net/minecraft/unmapped/C_jdubdjcx net/minecraft/world/gen/feature/MineshaftFeature
+	FIELD f_mfedizae CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_vqaicoqp mineshaftType Lnet/minecraft/unmapped/C_jdubdjcx$C_apnarkxp;
+	METHOD <init> (Lnet/minecraft/unmapped/C_quklzgtb$C_dnbqyhbk;Lnet/minecraft/unmapped/C_jdubdjcx$C_apnarkxp;)V
+		ARG 1 settings
+		ARG 2 mineshaftType
+	METHOD m_bgisrppb addPieces (Lnet/minecraft/unmapped/C_jieseoeb;Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_quklzgtb$C_ooyswlix;)V
+		ARG 1 collector
+		ARG 2 pos
+		ARG 3 context
+	METHOD m_dawtixdv (Lnet/minecraft/unmapped/C_jdubdjcx;)Lnet/minecraft/unmapped/C_jdubdjcx$C_apnarkxp;
+		ARG 0 feature
+	METHOD m_jsdevvpf (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
+		ARG 0 instance
+	METHOD m_vvalexls (Lnet/minecraft/unmapped/C_hynzadkk;Lnet/minecraft/unmapped/C_quklzgtb$C_ooyswlix;Lnet/minecraft/unmapped/C_jieseoeb;)V
+		ARG 3 collector
+	CLASS C_apnarkxp MineshaftType
```

Comment:
```suggestion
	CLASS C_apnarkxp Type
```
Redundant to have MineshaftFeature$MineshaftType.
Also update the field in the outer class and its initializer